.github/workflows/daily.yml

```diff
@@ -282,6 +282,76 @@ jobs:
       if: true && !contains(github.event.inputs.skiptests, 'cluster')
       run: ./runtest-cluster --config io-threads 4 --config io-threads-do-reads yes ${{github.event.inputs.cluster_test_args}}
 
+  test-ubuntu-reclaim-cache:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'workflow_dispatch' || (github.event_name != 'workflow_dispatch' && github.repository == 'redis/redis')) &&
+      !contains(github.event.inputs.skipjobs, 'reclaimcache')
```

Comment:
if we do that, we should add a `reclaimcache` part to the `skipjobs` defaults at the top of the file.
but maybe we should make it more generic so that other similar tests can be added in the future.
we could make it more generic by naming it `specific`, so we can choose to skip or not to skip all specific tests (i.e. ones that don't just re-run the whole test suite on various combinations).

another alternative is to move it to a separate (new) yaml file (with a daily cron trigger), but then we won't be able to trigger it with our workflow dispatch on random branches and PRs.

i'm leaning towards just renaming it.